test/test_xcpp_kernel.py

```diff
@@ -135,23 +134,36 @@ class XCppTests(jupyter_kernel_test.KernelTests):
         ]
 
 
-def test_notebook():
-    # Execute the notebook
-    executed_notebook = pm.execute_notebook(
-        'Notebooks/Untitled.ipynb',
-        'Notebooks/Untitled_output.ipynb',
-        log_output=True,
-        kernel_name='xcpp20'
-    )
-
-    # Read the executed notebook
-    with open('Notebooks/Untitled_output.ipynb') as f:
-        nb = nbformat.read(f, as_version=4)
-
-    # Get the output of the third cell
-    output = nb.cells[2].outputs[0]['text']
-
-    assert output == '12'
+    class XCppNotebookTests(unittest.TestCase):
+
+        inp = 'Notebooks/Untitled.ipynb'
```

Comment:
Now we need to make this a predefined list and have a test for each entry of that list. There should be some way to add programmatically new tests.